main.go

```diff
@@ -283,6 +285,26 @@ func main() {
 			ctx.String(err.Status(), err.Error())
 		}
 	})
+	router.GET("/config", func(ctx *gin.Context) {
```

Comment:
I don't think that the client should rely on the backend to know if it should be running locally.